SCons/Taskmaster/Job.py

```diff
@@ -84,6 +83,7 @@ def __init__(self, num, taskmaster):
         class can't do it, it gets reset to 1.  Wrapping interfaces that
         care should check the value of 'num_jobs' after initialization.
         """
+        from SCons.Script import GetOption
```

Comment:
Imports at top, maybe?

Reply:
Sigh.  Yeah, I've run into that a number of times.  Add a pylint annotation for that maybe, then?.
```py
# pylint: disable=import-outside-toplevel
```